src/common/low_precision_transformations/src/network_helper.cpp

```diff
@@ -1124,15 +1124,17 @@ std::shared_ptr<opset1::FakeQuantize> NetworkHelper::updateFakeQuantize(
     float min,
     float max,
     const bool replace) {
-    auto newMin = std::make_shared<opset1::Constant>(fq->get_output_element_type(0), Shape{}, min);
-    auto newMax = std::make_shared<opset1::Constant>(fq->get_output_element_type(0), Shape{}, max);
+    auto newOutMin = std::make_shared<opset1::Constant>(fq->get_output_element_type(0), Shape{}, min);
+    auto newOutMax = std::make_shared<opset1::Constant>(fq->get_output_element_type(0), Shape{}, max);
+    auto newInMin = getOutputWithSingleConsumer(fq->get_input_node_shared_ptr(1));
+    auto newInMax = getOutputWithSingleConsumer(fq->get_input_node_shared_ptr(2));
 
     std::shared_ptr<opset1::FakeQuantize> newFQ = std::make_shared<ngraph::op::TypeRelaxed<opset1::FakeQuantize>>(
             fq->input_value(0),
-            fq->input_value(1),
-            fq->input_value(2),
-            newMin->output(0),
-            newMax->output(0),
+            newInMin,
+            newInMax,
+            newOutMin->output(0),
+            newOutMax->output(0),
```

Comment:
1. this update is not covered by tests
2. `NetworkHelper::updateFakeQuantize` is used only if `IntervalsAlignmentAttribute` instances exist. The method is not exeucuted for the model from the ticket.

Reply:
The method is not executed for the model from the ticket but we also need to change it to avoid fusing issues. Thanks for the comment, this case has been covered by tests too.